test/system_tests/test_io.py

```diff
@@ -0,0 +1,94 @@
+import festim as F
+
+import dolfinx
+import mpi4py.MPI as MPI
+
+
+def test_out_in():
```

Comment:
Think we could improve the name of this one 😅 , or at least add some more docs underneath

Reply:
Meh couldn't find a better name sorry

Reply:
Meh couldn't find a better name sorry